generate.sh

```diff
@@ -22,3 +22,27 @@ rm generated/.gitignore
 rm -r ./docs/Model
 mv -f generated/docs/Model ./docs/Model
 rm -rf generated/docs/
+
+add_passage_version_header() {
+  local header_comment="// Add Passage version header"
+
+  if grep -q "$header_comment" generated/lib/HeaderSelector.php; then
+    echo "Passage version header already exists"
+    return 0
+  fi
+
+  local package_name="passage-php"
+
+  sed -i "/return \$headers;/i \\
+        $header_comment\\
+        try {\\
+            \$packageVersion = \\\\Composer\\\\InstalledVersions::getPrettyVersion('passageidentity/$package_name');\\
+            \$headers['Passage-Version'] = '$package_name ' . \$packageVersion;\\
+        } catch (\\\\Throwable \$ignored) {\\
+            // Don't throw an exception if we can't determine the package version\\
```

Comment:
Without an exception this will quietly fail.

We may want to provide a message that we were not able to determine the package version. 

and/or

It may still be helpful to still set the header without the version number to at least track the package name and see the missing version in reports.

Thoughts?

Reply:
updated in [1a83700](https://github.com/passageidentity/passage-php/pull/69/commits/1a8370046e3e6a2b9431c45acb6627fbe626ea41)